src/jira/client/axios.ts

```diff
@@ -230,10 +262,9 @@ const instrumentRequest = (response) => {
  * @param {import("axios").AxiosError} error - The Axios error response object.
  * @returns {Promise<Error>} a rejected promise with the error inside.
  */
-const instrumentFailedRequest = (logger) => {
+const instrumentFailedRequest = () => {
 	return (error) => {
 		instrumentRequest(error?.response);
-		logger.error(error, "Error during Axios request");
```

Comment:
Removing extra logging from here. This log will be duplicated in the error handler